src/remotebuild/lib/cli.ts

```diff
@@ -39,6 +40,9 @@ class CliHelper {
                 // if version flag found, show version and exit
                 CliHelper.handleVersionFlag(args);
             })
+            .then(function(): Q.Promise<any> {
+                return telemetry.init("REMOTE_BUILD", require("../package.json").version);
```

Comment:
Just a thought: Should the ID be "TACO/remotebuild"? Or should this be treated as a separate app?
@digeff